.github/ISSUE_TEMPLATE/good_first_issue.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Good first issue
+about: (Regular devs only): Suggest a new good first issue
+title: ''
+labels: good first issue
+assignees: ''
+
+---
+
+The purpose of the `good first issue` label is to highlight which issues are suitable for a new contributor without a deep understanding of the codebase.
+
+Useful skills:
+
+(For example, “C++11 std::thread”, “Qt5 GUI and async GUI design” or “basic understanding of Bitcoin mining and the Bitcoin Core RPC interface”.)
+
+Want to work on this issue?
+
+You do not need to request permission to start working on this. It is encouraged (but not compulsory) to comment on the issue if you are planning to work on it. This will help other contributors monitor which issues are actively being addressed and is also an effective way to request assistance if and when you need it.
```

Comment:
nit: could use active voice rather than passive, e.g. "You are encouraged" or "We encourage you" rather than "It is encouraged (but not compulsory)".